### PR TITLE
README: Add notes about Quest for Glory character import

### PR DESCRIPTION
**Pull request by raziel-**
Taken mostly from the wiki and from criezy´s latest additions

### CODE REVIEW

File: README [hunk 1]
```diff
@@ -935,8 +937,32 @@ In addition, you will need to rename disk image 2 to disk1,
...
+For example, a character can be imported from Quest for Glory I directly
+to Quest for Glory III without having to necessarily play
+Quest for Glory II.
+
+Characters cannot be imported from future games to past games, nor can
+a character be imported to the same game that was just completed.
+In other words, a character from Quest for Glory II cannot be imported
+into Quest for Glory II.
+
+If you want to use a saved character from the original sierra interpreter,
```
Comment (bonki) [inline README]: s/sierra/Sierra/ ?

Comment (raziel-) [reply README]: Yes, there is always one thing i miss :-)

**Comment by kintera** [comment]
Please remember quest for glory 4 has a bug in some versions that a qfg4
character can he imported back in to quest for glory 4, and 3's import is
buggy and will not always select the correct class on import.

**Comment by sev-** [comment]
Thanks, merging. Good stuff.

**Pull request merged**